docs/includes/topic_ha-modes.rst

```diff
@@ -1,23 +1,103 @@
-.. _ha-mode:
+:orphan: true
 
 HA mode
-```````
+=======
 
-- ``f5_ha_type``: Defines the high availability (HA) mode used by the BIG-IP®.
+Overview
+--------
 
-    * ``standalone``: Single BIG-IP® device; no HA.
-    * ``pair``: Active/standby two device HA.
-    * ``scalen``: Active device cluster.
+:term:`HA`, or, 'high availability', mode refers to high availability of the BIG-IP® device(s). The F5® agent can configure BIG-IP to operate in :term:`standalone`, :term:`pair`, or :term:`scalen` mode. The F5 agent uses :ref:`replication mode <Managing BIG-IP Clusters>` to configure LBaaS objects on HA devices in real time.
+
+Use Case
+--------
+
+High availability modes provide redundancy, helping to ensure service interruptions don't occur if a device goes down.
+
+* :term:`standalone` mode utilizes a single BIG-IP device; here, 'high availability' means that BIG-IP core services are up and running, and VLANs are able to send and receive traffic to and from the device.
+
+* :term:`pair` mode requires two (2) BIG-IP devices.
```

Comment:
Suggest: `pair` mode requires two (2) BIG-IP devices and provides active/standby operation. When an event occurs that prevents the 'active' BIG-IP device from processing network traffic, the 'standby' device immediately begins processing that traffic so users experience no interruption in service. There is no loss in performance since the standby device takes over the entire traffic load.